src/mcts.rs

```diff
@@ -288,6 +331,16 @@ impl<'a> Searcher<'a> {
         (pv, score)
     }
 
+    fn get_best_move(&self) -> Move {
+        let idx = self.tree.get_best_child(self.tree.root_node());
+        let action = self.tree.edge(self.tree.root_node(), idx);
+        Move::from(action.mov())
+    }
+
+    fn get_cp(&self, score: f32) -> f32 {
```

Comment:
This doesn't need to be a method. Remove the `&self` and you can call it with `Searcher::get_cp(score)`.